dextra-components/src/stories/Bar.stories.ts

```diff
@@ -1,16 +1,61 @@
 import { Meta } from "@storybook/web-components";
-import { html } from "lit";
-import "../components/marks/Histogram/src/Histogram.ts";
+import { html } from "lit"; 
+import "../components/marks/Bar/src/Bar.ts";
 import "../components/Plot/src/Plot.ts";
 
 export default {
   title: "Marks",
-};
+}
 
-export const Histogram = () => {
-  return html` <osl-plot scheme="inferno"
-  data="https://d3uldu0bz6pkei.cloudfront.net/data-query/646fd0f4e06ddb00081c002a"
-  >
-    <osl-histogram-plot x="MedianHouseholdIncome" fill="forestgreen"></osl-histogram-plot>
-  </osl-plot>`;
-};
+export const Bar = () => {
+  return html`
+  <osl-plot
+      width = 1000
+      height = 500
+      marginLeft = 180
+      data="http://localhost:6006/data/pesticide.json"
+      x_type="sqrt"     
```

Comment:
Seeing this here, we might want to call this `xAxisScaling` in the API?

Reply:
[21aca46](https://github.com/open-spatial-lab/dextra/pull/4/commits/21aca468057fb1f2d8a798507653fba84b1df7bd)
changed name to xAxisScaling